core/sql/executor/ExExeUtilLoad.cpp

```diff
@@ -3288,6 +3288,45 @@ short ExExeUtilLobExtractTcb::work()
   return 0;
 }
 
+
+short ExExeUtilLobExtractLibrary(ExeCliInterface *cliInterface,char *libHandle, char *cachedLibName,ComDiagsArea *toDiags)
+{
+  char buf[1000];
```

Comment:
I assume that cachedLibName won't be greater than 1000 bytes.  You could:  char buf[strlen(cachedLibName) + strlen(libHandle) + 200] so you have a better indication of the size.